src/test/func/malloc/malloc.cc

```diff
@@ -36,12 +36,19 @@ void check_result(size_t size, size_t align, void* p, int err, bool null)
     failed = true;
   }
   const auto alloc_size = our_malloc_usable_size(p);
-  const auto expected_size = round_size(size);
+  auto expected_size = round_size(size);
 #ifdef SNMALLOC_PASS_THROUGH
   // Calling system allocator may allocate a larger block than
   // snmalloc. Note, we have called the system allocator with
   // the size snmalloc would allocate, so it won't be smaller.
   const auto exact_size = false;
+  // We allocate 1 byte for 0-sized allocations (and so round_size will tell us
+  // that the minimum size is 1), but the system allocator may return a 0-sized
+  // allocation.
```

Comment:
```suggestion
  // We allocate MIN_ALLOC_SIZE byte for 0-sized allocations (and so round_size will tell us
  // that the minimum size is MIN_ALLOC_SIZE), but the system allocator may return a 0-sized
  // allocation.
```
Probably won't pass clangformat with my change.